contracts/DependencyRegistryV0.sol

```diff
@@ -829,4 +829,14 @@ contract DependencyRegistryV0 is
         OwnableUpgradeable._transferOwnership(newOwner);
         adminACLContract = IAdminACLV0(newOwner);
     }
+
+    /**
+     * Helper for calling `BytecodeStorageReaderV1` external library reader method,
+     * added for gas-optimization purposes.
```

Comment:
nit: bytecode size reduction, not gas (this comment applies to all contracts in this PR)

Reply:
Great catch – updated, here and throughout 🙏